event/generate.go

```diff
@@ -116,6 +116,9 @@ func main() {
 		"visibilitychange":         "VisibilityChange",
```

Comment:
This is out of scope, but it'd be nice to document `nameMap` to explain what its purpose is. It's easy to forget...

IIRC, it's to generate Go names that are compatible with Go style on [MixedCaps](https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps) and [initialisms](https://github.com/golang/go/wiki/CodeReviewComments#initialisms). But I can't remember if that's exactly it, or there's another reason for this.